AsyncExpiringLazy/AsyncExpiringLazy.csproj

```diff
@@ -14,6 +14,7 @@
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/filipw/async-expiring-lazy.git</RepositoryUrl>
     <PackageTags>async;lazy;expiring</PackageTags>
+    <Version>1.1.1</Version>
```

Comment:
this is not needed when `<VersionPrefix>` is there already

Reply:
Good spot, Visual studio auto added it when i updated the version through the properties window UI. 

I have removed it